internal/pkg/list/list_test.go

```diff
@@ -36,7 +36,7 @@ func TestList_JobListWriter(t *testing.T) {
 			inputAppName:   mockAppName,
 			inputWriteJSON: false,
 
-			wantedContent: "Name                Type\n--------            -------------\nbadgoose            Scheduled Job\nfarmer              Scheduled Job\n",
+			wantedContent: "Name                Type\n----                ----\nbadgoose            Scheduled Job\nfarmer              Scheduled Job\n",
```

Comment:
Can we make the content easier to read as well? like https://github.com/aws/copilot-cli/pull/1705/files#diff-2cf2608db8d11464d4ffcc84647f98c75de2a474d1c6deadcedeae0806b4c0feR288-R316

Reply:
yup, done.